docs/application/dotnet/guides/app-management/app-controls.md

```diff
@@ -506,8 +499,9 @@ application requesting a launch of your application must either specify
 your application ID (for an explicit launch) or have the same operation
 value and applicable URI and MIME type information (for an implicit
 launch).
+You can define privielges to restrict launching for applications having certain defined privileges.
```

Comment:
You can define privielges to restrict launching for applications having certain defined privileges.

Please rephrase to:

You can define privileges to restrict launching of applications.


The content having certain predefined privileges is not clear.

Reply:
done

Reply:
I've prepared a new patch. Please review it.